build.sbt

```diff
@@ -304,6 +310,22 @@ val jsdocs = project
   )
   .enablePlugins(ScalaJSPlugin, ScalaJSBundlerPlugin)
 
+val jswebsitedocs = project
```

Comment:
I think the `jsdocs` project was being used in both testing mdoc functionality, and for the website. The `jsdocs` project remains valid, because I assume that we wish to preserve the existing suite of js tests for mdoc. 

The new project is used solely for website generation...

I think the short answer then is a seperation of concerns between mdocs own website, and testing it's JS functionality.